policies/access.md

```diff
@@ -0,0 +1,41 @@
+# Github Access Management Policy
+
+## Teams, not Individuals
+
+Access to repositories will be granted only to Github teams, not to individuals. A repo may have any number of teams added to it, with the appropriate access levels.
+
+## Team Structure
+
+Teams will be nested, using parent/child relationships, as needed. A child team will implicitly have any privileges its parent team does, so, the most deeply nested team should be the one with the most privileges.
+
+### Top-level teams
+
+Note: this list is intentionally not exhaustive.
+
+- [TAC](https://github.com/orgs/ossf/teams/tac)
+    This team is for current TAC members. Individuals should be added or removed from this team to reflect current membership.
+- [Staff](https://github.com/orgs/ossf/teams/staff)
+    This team is for OpenSSF staff members. Individuals on the top-level team are there as a catch-all, but may be moved to subteams as the need arises.
+  - [PMs](https://github.com/orgs/ossf/teams/pms)
+      This teams is for PMs, who often need Maintain or Admin access on repos.
+  - [Marketing](https://github.com/orgs/ossf/teams/marketing)
+- [Working Groups](https://github.com/orgs/ossf/teams/working-groups)
+    This is the parent team for Working Groups. Every WG should have a subteam contained within this one. All WG subteams must start with `wg-` for consistency.
+- [SIGs](https://github.com/orgs/ossf/teams/sigs)
+    This is the parent team for SIGs. Every SIG should have a subteam contained within this one. All SIG subteams must start with `sig-` for consistency.
+- [Projects](https://github.com/orgs/ossf/teams/projects)
+    This is the parent team for projects. Every project (eg scorecard, AO) should have a subteam contained within this one.
+    Teams for individual repositories go under here, which start with `repo-`, but team names may otherwise be unconstrained.
+
+## Github Org Membership
+
+ Membership in the Github org should be freely given - it inherently confers no permissions or privileges, only a badge on the user's profile if they enable it - and it _does_ allow for easier team management. Someone should only be removed from the org in extreme circumstances where their association with OpenSSF would be problematic, and people should be encouraged to remain in the org in perpetuity.
```

Comment:
Currently, it looks like members of https://github.com/ossf can create public and private repositories. We should probably audit https://github.com/organizations/ossf/settings/member_privileges before opening up membership as described in this document.

Reply:
It seems reasonable to me to restrict repo creation to OpenSSF staff and the TAC, if there’s agreement on that.

Reply:
This is NOT done correctly. I am not sure how to allow certain teams to create repos.
In https://github.com/organizations/ossf/settings/member_privileges it is not team-based. I would love a learning session on this from someone who knows how to do this.

Reply:
@hythloda the mechanism we're currently using is that only org owners can create repos, and all TAC members and some PMs are org owners. It indeed can't be team-based.